diff --git a/drivers/nvme/host/pci.c b/drivers/nvme/host/pci.c
index 989f31471da6..8691051a105b 100644
--- a/drivers/nvme/host/pci.c
+++ b/drivers/nvme/host/pci.c
@@ -3478,6 +3478,8 @@ static const struct pci_device_id nvme_id_table[] = {
        { PCI_DEVICE(0x1e4B, 0x1002),   /* MAXIO MAP1002 */
                .driver_data = NVME_QUIRK_BOGUS_NID, },
        { PCI_DEVICE(0x1e4B, 0x1202),   /* MAXIO MAP1202 */
+               .driver_data = NVME_QUIRK_BOGUS_NID, }
+       { PCI_DEVICE(0x1e4B, 0x1602),   /* MAXIO MAP1602 */
                .driver_data = NVME_QUIRK_BOGUS_NID, },
        { PCI_DEVICE(0x1cc1, 0x5350),   /* ADATA XPG GAMMIX S50 */
                .driver_data = NVME_QUIRK_BOGUS_NID, },